docs/03-tutorials/api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Use a custom domain to expose a service
+---
+
+This tutorial shows how to set up your custom domain and prepare a certificate to use the domain for exposing a service. The components used are Gardener [External DNS Management](https://gardener.cloud/docs/concepts/networking/dns-managment/#external-dns-management) and [Certificate Management](https://gardener.cloud/docs/concepts/networking/cert-managment/).
+
+To learn how to expose a service, go to [this](./apix-01-expose-service-apigateway.md) tutorial.
+
+## Prerequisites
+
+If you use a cluster not managed by Gardener, install the required components manually. Follow these steps:
+
+1. Create a Namespace. Run:
+
+   ```bash
+   kubectl create ns {NAMESPACE_NAME}
+   ```
+
+2. Download the [External DNS Management](https://github.com/gardener/external-dns-management) and [Certificate Management](https://github.com/gardener/cert-management) projects locally.
```

Comment:
I lack version information. Is using `master` safe ?

Reply:
Also we should provide the git clone commands to have this guide user friendly, because these helm charts are not available on any Helm repo.

Reply:
@Tomasz-Smelcerz-SAP I tested these components using master branch. They are connected with each other as cert-management requires dns-management. Here is quote from their README:
```The cert-controller-manager relies on DNS challenges (ACME only) for validating the domain names of the certificates. For this purpose it creates DNSEntry custom resources (in a possible separate dns cluster) to be handled by the compagnion dns-controller-manager from external-dns-management.```
Maybe we should advise users to simply install both of them from the latest release?

@cnvergence Agree, it would be quite handy.